internal/scaffold/ansible/dockerfilehybrid.go

```diff
@@ -100,4 +100,4 @@ COPY playbook.yml ${HOME}/playbook.yml[[ end ]]
 ENTRYPOINT ["/tini", "--", "/usr/local/bin/entrypoint"]
 
 USER ${USER_UID}
-`
+` // nolint:golint
```

Comment:
Could we ignore/exclude this file in another way? 
WDYT about add the config as described [here](https://github.com/golangci/golangci-lint#config-file)? Then is possible to `skip-dirs` and `skip-files`

Reply:
Solved 👍